### PR TITLE
[Typo] Duffel Bag

### PR DESCRIPTION
**Pull request by ShizCalev**
Corrects dufflebag to duffel bag. 
Adds new subtypes because having the same var-edited item across every map is weird.

### CODE REVIEW

**Comment by Chronitonity** [comment]
Dufflebag isnt a word, duffel bag is the correct term.

**Comment by ShizCalev** [comment]
Duffle is an acceptable alternative spelling to Duffel, but I too prefer Duffel.

**Comment by Chronitonity** [comment]
Oh nevermind i cant read

**Comment by ShizCalev** [comment]
Incase people want the option, I'll make another commit and just revert it if they don't like duffel.

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 1]
```diff
@@ -19702,10 +19702,10 @@
 /obj/item/weapon/reagent_containers/glass/bottle/facid{
 	pixel_x = 7
 	},
-/obj/item/weapon/storage/backpack/dufflebag/sec{
+/obj/item/weapon/storage/backpack/duffelbag/sec{
```
Comment (lzimann) [inline _maps/map_files/Deltastation/DeltaStation2.dmm]: Make this a proper subtype please, map edited things are awful

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 2]
```diff
@@ -1847,10 +1847,10 @@
 	pixel_x = -28
 	},
 /obj/structure/table,
-/obj/item/weapon/storage/backpack/dufflebag/sec{
+/obj/item/weapon/storage/backpack/duffelbag/sec{
 	contents = newlist(/obj/item/weapon/scalpel,/obj/item/weapon/hemostat,/obj/item/weapon/retractor,/obj/item/weapon/cautery,/obj/item/weapon/circular_saw,/obj/item/weapon/surgical_drapes,/obj/item/clothing/mask/surgical);
-	desc = "A large dufflebag for holding extra supplies - this one has a material inlay with space for various sharp-looking tools.";
```
Comment (lzimann) [inline _maps/map_files/MetaStation/MetaStation.dmm]: Same

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 3]
```diff
@@ -15127,11 +15127,11 @@
 /obj/item/weapon/storage/secure/briefcase{
 	contents = newlist(/obj/item/clothing/suit/armor/vest,/obj/item/weapon/gun/ballistic/automatic/pistol,/obj/item/weapon/suppressor,/obj/item/weapon/melee/classic_baton/telescopic,/obj/item/clothing/mask/balaclava,/obj/item/bodybag,/obj/item/weapon/soap/nanotrasen)
 	},
-/obj/item/weapon/storage/backpack/dufflebag{
+/obj/item/weapon/storage/backpack/duffelbag{
 	contents = newlist(/obj/item/clothing/under/lawyer/blacksuit,/obj/item/clothing/accessory/waistcoat,/obj/item/clothing/suit/toggle/lawyer/black,/obj/item/clothing/shoes/laceup,/obj/item/clothing/gloves/color/black,/obj/item/clothing/glasses/sunglasses,/obj/item/clothing/head/fedora);
-	desc = "A large dufflebag for holding extra things. There is a NanoTrasen logo on the back.";
-	icon_state = "duffle-syndieammo";
-	item_state = "duffle-syndieammo"
+	desc = "A large duffel bag for holding extra things. There is a NanoTrasen logo on the back.";
```
Comment (lzimann) [inline _maps/map_files/MetaStation/MetaStation.dmm]: Same

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 4]
```diff
@@ -82886,10 +82886,10 @@
 /area/shuttle/abandoned)
 "cYD" = (
 /obj/structure/table,
-/obj/item/weapon/storage/backpack/dufflebag/med{
+/obj/item/weapon/storage/backpack/duffelbag/med{
 	contents = newlist(/obj/item/weapon/scalpel,/obj/item/weapon/hemostat,/obj/item/weapon/retractor,/obj/item/weapon/cautery,/obj/item/weapon/circular_saw,/obj/item/weapon/surgicaldrill,/obj/item/weapon/razor);
-	desc = "A large dufflebag for holding extra medical supplies - this one seems to be designed for holding surgical tools.";
-	name = "surgical dufflebag";
```
Comment (lzimann) [inline _maps/map_files/MetaStation/MetaStation.dmm]: Same

**Review by lzimann — CHANGES_REQUESTED** [review]
I didn't comment on all, but please make the map edited duffels to be a proper subtype while you are at it

**Comment by ShizCalev** [comment]
@lzimann Done

**Review by lzimann — APPROVED** [review]

**Pull request merged**